src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -105,19 +109,76 @@ const EnhancedTableHead = (props: EnhancedTableProps) => {
 };
 
 const CCPTabPanel = ({ eventId }: { eventId: string }) => {
-  const [orderBy, setOrderBy] = React.useState<string>('name');
   const classes = useStyles();
+  const [orderBy, setOrderBy] = React.useState<string>('name');
   const [order, setOrder] = React.useState<Order>('asc');
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(
+    null
+  );
+  const [openConfirmDelete, setOpenConfirmDelete] = React.useState<boolean>(
+    false
+  );
+  const [selectedCCP, setSelectedCCP] = React.useState<CCP | null>(null);
   const { data } = useQuery(GET_CCPS_BY_EVENT_ID, { variables: { eventId } });
 
   const rows = data ? data.collectionPointsByEvent : [];
 
+  const handleCloseMenu = () => {
+    setAnchorEl(null);
+    setSelectedCCP(null);
+  };
+
+  const handleCloseConfirmDelete = () => {
+    handleCloseMenu();
+    setOpenConfirmDelete(false);
+  };
+
+  //  Writing to cache when deleting ccp
+  const [deleteCCP] = useMutation(DELETE_CCP, {
+    update(cache, { data: { deleteCollectionPoint } }) {
+      if (!deleteCollectionPoint) {
+        return;
+      }
+      let { collectionPointsByEvent } = cache.readQuery<any>({
+        query: GET_CCPS_BY_EVENT_ID,
```

Comment:
If/when we add a `GET_ALL_CCPS` query, we should probably update the cache for that query here too.